support/fixtures/factories/contxtUserPermissions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+const times = require('lodash.times');
+
+factory
+  .define('contxtUserPermissions')
+  .option('fromServer', false)
+  .attrs({
+    applicationsExplicit: () =>
+      times(
+        faker.random.number({ min: 0, max: 15 }),
+        () => factory.build('contxtApplication').id
+      ),
+    applicationsImplicit: () =>
+      times(
+        faker.random.number({ min: 0, max: 15 }),
+        () => factory.build('contxtApplication').id
+      ),
+    roles: () =>
+      times(
+        faker.random.number({ min: 0, max: 15 }),
+        () => factory.build('contxtRole').id
+      ),
+    stacksExplicit: () =>
+      times(
+        faker.random.number({ min: 0, max: 15 }),
+        () => factory.build('contxtStack').id
+      ),
+    stacksImplicit: () =>
+      times(
+        faker.random.number({ min: 0, max: 15 }),
+        () => factory.build('contxtStack').id
+      ),
+    userId: () => faker.random.uuid()
```

Comment:
We are using something like `test|${faker.internet.password()` elsewhere in the fixtures -- i'd recommend just using the user fixture

Reply:
I can't believe I missed this :man_facepalming: